src/components/views/read/cvForm/Education.tsx

```diff
@@ -66,7 +66,7 @@ const Education: FC<Props> = ({ nextStep, prevStep, onError }) => {
 
   const [showFields, setShowFields] = useState<{ [key: string]: boolean }>({});
   const [errors, setErrors] = useState<{ [key: string]: string }>({});
-  const [isValid, setIsValid] = useState(true);
+  let isValid: boolean;
```

Comment:
You don't need this at the top level.
You use it in only 2 function: 
- handleDone: in which you never read it. so just remove it from there.
- handleNext: in which you could declare it at the start of the function with initial value of `true` and if you have an error you set it to false.

Reply:
Done.